pkg/volume/validation/pv_validation.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validation
+
+import (
+	"github.com/golang/glog"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/v1"
+	"k8s.io/kubernetes/pkg/volume"
```

Comment:
So when I was talking to you today, what I was assuming would be that *all* mount options validation would be split out into it's own package, with no dependency on the probe plugins mechanism.  I don't see why you need to probe plugins - the API defines the structs, those always exist, and the validation for those should be totally deterministic.

I'd be ok with that change - I don't think this is the correct way to include validation.  SupportsMountOption should be code that is in a separate package (no dependencies), and then referenced both by each mount plugin and referenced by this validation code.  Does that make sense?